### ISSUE TITLE
Permettre à l'utilisateur de recevoir un mail lorsque il essaie de s'inscrire avec une adresse déjà utlisée 

### ISSUE THREAD

**Issue by florianpanchout** [betagouv/recosante-mail#54]
Si l'utilisateur entre une adresse mail déjà associée à un utilisateur on lui propose de recevoir un mail avec un lien vers son profil.

La partie front est déjà en place, il manque : 
- [ ] Le endpoint qui déclenche l'envoi du mail
- [ ] Le template du mail

**Comment by florianpanchout** [comment]
Le template du mail est ici : https://my.sendinblue.com/camp/template/1454/message-setup

**Comment by l-vincent-l** [comment]
Le mail s’envoie si une personne essaie de se réinscrire

**Comment by florianpanchout** [comment]
Sur quel endpoint ?

**Comment by l-vincent-l** [comment]
Pour l’instant le mail s’envoie automatiquement à la suite d’une tentative d’inscription avec un mail déjà existant.

On pourrait pour la suite faire un endpoint spécial, mais il faudrait pour ça générer un identifiant temporaire lié au mail pour ne pas faire fuiter le `uid` existant à une personne qui n’est pas forcément la personne concernée.

**Comment by l-vincent-l** [comment]
Oui, je viens de penser qu’on n’a pas besoin de ça, le endpoint peut accepter un mail.
Je fais ça

**Comment by l-vincent-l** [comment]
J’ai ajouté le endpoint.
Le détail est dans le premier commentaire de l’issue

**Comment by florianpanchout** [comment]
Merci ! C'est intégré en front. Par contre je ne reçois pas d'email. C'est parce que l'on est sur le staging ?
